Applications/Spire/Source/KeyBindings/TaskKeyBindingsTableView.cpp

```diff
@@ -0,0 +1,147 @@
+#include "Spire/KeyBindings/TaskKeyBindingsTableView.hpp"
+#include "Nexus/Definitions/OrderType.hpp"
+#include "Nexus/Definitions/Quantity.hpp"
+#include "Nexus/Definitions/Region.hpp"
+#include "Nexus/Definitions/Security.hpp"
+#include "Nexus/Definitions/Side.hpp"
+#include "Nexus/Definitions/TimeInForce.hpp"
+#include "Spire/KeyBindings/DestinationItemDelegate.hpp"
+#include "Spire/KeyBindings/KeySequenceEditor.hpp"
+#include "Spire/KeyBindings/KeySequenceItemDelegate.hpp"
+#include "Spire/KeyBindings/NameItemDelegate.hpp"
+#include "Spire/KeyBindings/OrderTypeItemDelegate.hpp"
+#include "Spire/KeyBindings/QuantityItemDelegate.hpp"
+#include "Spire/KeyBindings/SecurityInputItemDelegate.hpp"
+#include "Spire/KeyBindings/SideItemDelegate.hpp"
+#include "Spire/KeyBindings/TimeInForceItemDelegate.hpp"
+#include "Spire/Spire/Dimensions.hpp"
+#include "Spire/Spire/Utility.hpp"
+#include "Spire/Ui/Ui.hpp"
+
+using namespace Beam;
+using namespace Nexus;
+using namespace Spire;
+using ValidSequence = KeySequenceEditor::ValidKeySequence;
+
+namespace {
+  const auto& FUNCTION_KEYS() {
+    static auto function_keys = std::set<Qt::Key>({
```

Comment:
Don't use `std::set` or `std::map` unless you need the sorted order.
Prefer hashes to trees.